Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -557,6 +563,14 @@ extension Docc {
             
         }
 
+        /// A user-provided value that is true if the user enables experimental support for code block annotation.
+        ///
+        /// Defaults to false.
+        public var enableExperimentalCodeBlock: Bool {
```

Comment:
nit: Like above, as a public API I feel like this name is missing a word somewhere. Right now, this reads as "is code blocks enabled" which is misleading and isn't what the value represents.

One way to clarify what this represents could be to add a word like "annotations" or "extensions" or similar.